TypewiseAlert/XmlData.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.Xml;
+
+namespace TypewiseAlert
+{
+    public class XmlData
+    {
+        public List<CoolingTypeModel> GetCoolingTypeData()
+        {
+            try
+            {
+                List<CoolingTypeModel> coolingTypeList = new List<CoolingTypeModel>();
+                CoolingTypeModel coolingType = null;
+
+                XmlDocument xmlDocument = new XmlDocument();
+                xmlDocument.Load(string.Format("{0}/{1}", AppDomain.CurrentDomain.BaseDirectory, "CoolingTypeFile.xml"));
+
+                XmlNodeList nodeList = xmlDocument.DocumentElement.SelectNodes("/CoolingTypeList/CoolingType");
+                
+                foreach (XmlNode xmlNode in nodeList)
+                {
+                    coolingType = new CoolingTypeModel();
+                    coolingType.CoolingType = Convert.ToInt32(xmlNode.SelectSingleNode("CoolingType").InnerText);
+                    coolingType.LowerLimit = Convert.ToDouble(xmlNode.SelectSingleNode("LowerLimit").InnerText);
+                    coolingType.Upperlimit = Convert.ToDouble(xmlNode.SelectSingleNode("Upperlimit").InnerText);
+
+                    coolingTypeList.Add(coolingType);
+                }
+                
+                return coolingTypeList;
+            }
+            catch(Exception)
```

Comment:
catching general Exception type and ignoring it isn't a good idea. Have a defined behavior in exception cases and test for it

Reply:
Hi Sudeep. Thank you for your feedback. Will update the code based on your feedback.
Thank you.